tests/foreman/ui/test_reporttemplates.py

```diff
@@ -414,9 +415,11 @@ def test_positive_schedule_generation_and_get_mail(
                 'email_to': 'root@localhost',
             },
         )
-    file_path = PurePath('/tmp/').joinpath(f'{gen_string("alpha")}.json')
+    randstring = f'{gen_string("alpha")}'
+    file_path = PurePath('/tmp/').joinpath(f'{randstring}.json')
```

Comment:
any specific reason using `/tmp/`  instead of `robottelo_tmp_dir` here?

Reply:
Its ultimately referring to a directory on the sat host, not locally. robottelo_tmp_dir is being used below for the `local_file`. `PurePath`  lets you take advantage of pathlib, but without using a path object that will interact with the local filesystem.